libs/report/committers/src/lib/committers.ts

```diff
@@ -23,12 +23,12 @@ export const reportCommittersCommand: CommandModule<object, Options> = {
     beforeDate: {
       alias: 's',
```

Comment:
I think it would make sense to also swap the aliases (I assume `s` stands for "start" and `e` for "end").

Reply:
yeah agreed but release a major for this 😆 sounds funny